tests/elm/ElmLintBearTest.py

```diff
@@ -0,0 +1,6 @@
+from bears.elm.ElmLintBear import ElmLintBear
+from tests.LocalBearTestHelper import verify_local_bear
```

Comment:
Please use `coalib.testing.LocalBearTestHelper` over `tests.LocalBearTestHelper`, it's deprecated.